<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RAISRWidget</class>
 <widget class="QWidget" name="RAISRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lned_input"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_selfile">
       <property name="text">
        <string>选择输入图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_doraisr">
       <property name="text">
        <string>执行升采样</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="lab_input">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_output">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
